crates/op-rbuilder/src/builders/builder_tx.rs

```diff
@@ -276,11 +271,20 @@ impl BuilderTxBase {
     }
 }
 
-pub(super) fn get_nonce(
+pub fn get_nonce(
     db: &mut State<impl Database>,
     address: Address,
 ) -> Result<u64, BuilderTransactionError> {
     db.load_cache_account(address)
         .map(|acc| acc.account_info().unwrap_or_default().nonce)
         .map_err(|_| BuilderTransactionError::AccountLoadFailed(address))
 }
+
+pub fn get_balance(
```

Comment:
```suggestion
pub(crate) fn get_balance(
```

Reply:
flashtestations crate can not access this function, need to reexport